mybatis-spring-boot-starter/pom.xml

```diff
@@ -27,9 +27,25 @@
   <name>mybatis-spring-boot-starter</name>
   <dependencies>
     <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-jdbc</artifactId>
+    </dependency>
+    <dependency>
       <groupId>org.mybatis.spring.boot</groupId>
       <artifactId>mybatis-spring-boot-autoconfigure</artifactId>
-      <version>1.2.0-SNAPSHOT</version>
+      <version>${project.version}</version>
```

Comment:
@snicoll , thanks for review !!
I've fixed your comment via 786af69.
If possible, please check again.